src/utils/tables.js

```diff
@@ -39,7 +39,7 @@ export function getColumns(tableConfig, classes, data, externalLinkIcon, linkto,
                   {' '}
                   {column.dataFromRoot ? data[column.dataField]
                     : (column.formatBytes ? formatBytes(value)
-                      : column.dataField === 'num_subjects'
+                      : (column.dataField === 'num_subjects' || column.dataField === 'numberOfCases')
```

Comment:
The implementation fixes the bug correctly.  But will be possible to change the hardcode? If that takes long would not be fix in this release, can we have a ticket in the parking lot to keep the track of this?

Reply:
No we dont have any ticket. and if we change it in backend API we wont need this change. 
<img width="1086" alt="Screen Shot 2020-12-03 at 1 51 47 PM" src="https://user-images.githubusercontent.com/9453011/101074424-b12b1f00-356e-11eb-9a26-9329adb93699.png">